backend/README.md

```diff
@@ -30,13 +30,13 @@ There are suites covering all endpoints. Each suite should test all response sta
 A test suite for an endpoint should check for status and data to ensure no false positive responses are being created. 
 The simplest test looks like:
 
-```
+```typescript
```

Comment:
You can also just use `ts` for short :)

Reply:
Sounds good. We'll take note of that in the future we use code blocks in markdown. Thanks Matthew